fully styled section pt1

mykhaito opened the pull request.

Ustym-Bordun reviewed (commented):
Треба поправити :
1. В файлі base.css для body прописано font-family (з загальними стилями для body) І тому ми не дублюємо цю властивість .
2. Для тегу секції в файлі base.css є окремий клас .section . Якщо він підходить (а в вашому варіанті він підходить) то використовуємо його щоб не дублювати код) .
3. Для background-color  в секціях використовуємо ще один зарезервований клас (.accent-bg-color або .second-accent-bg-color) який є у файлі base.css . Щоб не дублювати код (і в разі потреби змінювати колір у всього сайту через декілька класів) .
4. Дів з класом .your-order-text-block є непотрібним . Все що він виконує це відпихається нижнім марженом він від наступної частинки . Цю ж функцію може виконувати і параграф з класом .your-order-text-block-paragraph 
5. Кольори там де можливо ми задаємо через зарезервований клас (наприклад .second-main-color) (Це реалізується добавлянням додаткового класу в розмітці)
Тобто для класу :
.your-order-text-block-header , .your-order-text-block-paragraph
6. Для класу .your-order-form-link-block можна не задавати flex-direction: column; а задати таку властивість як flex-wrap: wrap;  але тоді обовязково потрібно додати ще класам .your-order-orderform-block і .your-order-linkform-block властивість width: 100%; (щоб вони не прорвали наш дів) . Це не є помилкою але я б реалізував це момент так.

mykhaito commented:
-Implemented integration with base.css
-Added only 1 possible calc into   
.your-order-orderform-block,
  .your-order-linkform-block
 -picture is used
- fixed section usage

mykhaito commented:
+ Commit with animations, <script src="./js/call-back.js"></script> added to index

Ustym-Bordun reviewed (changes requested):
Потрібно поправити : 
1. Для класу your-order-text-block-header властивість width: 640px; на планшетній версії не потрібна
2. Для класу your-order-form-link-block властивість justify-content: center;  не потрібна
3. На десктопній версії для класів your-order-orderform-block і your-order-linkform-block формула для властивості width має бути такою ( width: calc((100% - 24px) / 2); )
Властивість flex: 1; для цих класів  не потрібна
4. Зображення мають бути у форматі png
5. Потрібно спочатку записати в css файл всі стилі для моб версії потім перезаписати їх через одне медіа правило для планшетів і потім перезаписати через ще одне медіа правило 
Щоб не плутатись у файлі і в каскадності
6. Через те що у ви звертаєтесь до медіа правил кілька разів у вас деякі властивості не правильно перезаписуються 
Напириклад : для класів your-order-orderform-block і your-order-linkform-block властивість на десктопну версію сторінки width: calc((100% - 24px) / 2);
7. Кнопка має бути поза межами форми.
8. Додати властивість outline: none; на кнопку.

mykhaito commented:
DONE

Ustym-Bordun reviewed (changes requested):
### Внести всі обговорені останні правки

Ustym-Bordun reviewed (approved):
### **Section your order done**
